paris-paristechprofile/week-2/js-racer-game-lab/Paris' Code/app.js

```diff
@@ -0,0 +1,58 @@
+window.onload = function () {
+  player = {
+    blue: {
+      div: document.querySelector(".blueBox"),
+      name: "Blue"
+    },
+    red: {
+      div: document.querySelector(".redBox"),
+      name: "Yellow"
+    },
+  };
+};
+
+//activate button listening
+const go = document.querySelector(".ready").addEventListener("click", function() {
+//show a user a GO!
+    document.getElementsByTagName("h2")[0].innerText = "GO!";
+});
+
+//determine blueBox right boundary
+	var blueBoxRight = parseInt(document.querySelector('.blueBox').getBoundingClientRect().right);
+//determine redBox right boundary
+	var redBoxRight = parseInt(document.querySelector('.redBox').getBoundingClientRect().right);
+//determine finish line 
+	let finishLine = parseInt(document.querySelector('.finish').getBoundingClientRect().left);
+
+let blueBox = document.querySelector('.blueBox');
+let redBox =  document.querySelector('.redBox') //do the same for red
+  	window.addEventListener('keydown', moveBox); // this is comprised of an event and a callback function 
+  	var blueLeft = 0;
```

Comment:
If you are using Let in your document you should not be using var as well. It is not a good practice to mix the two. Moving forward try to stick to one or the other.